### PR TITLE
fixes foldering of the gpt2 minio notebook

### LINKED ISSUE #4080: typo in "Pretrained GPT2 Model Deployment Example" notebook
## Describe the bug
I think there is a typo in the following notebook [Pretrained GPT2 Model Deployment Example](https://docs.seldon.io/projects/seldon-core/en/latest/examples/triton_gpt2_example.html)

```
!mc mb minio-seldon/onnx-gpt2 -p
!mc cp ./model.onnx minio-seldon/onnx-gpt2/gpt2/1/
```

Should be
```
!mc mb minio-seldon/onnx-gpt2 -p
!mc cp ./model.onnx minio-seldon/onnx-gpt2
```

as in the:

```
%%writefile gpt2-deploy.yaml
apiVersion: machinelearning.seldon.io/v1alpha2
kind: SeldonDeployment
metadata:
  name: gpt2
spec:
  predictors:
  - graph:
      implementation: TRITON_SERVER
      logger:
        mode: all
      modelUri: s3://onnx-gpt2
      envSecretRefName: seldon-init-container-secret
      name: gpt2
      type: MODEL
    name: default
    replicas: 1
  protocol: kfserving
```

In the current format in the website init container was not able to find the model from Minio but putting the model in the root directory of the minio bucket solved the problem.

### PR DESCRIPTION
**Pull request by saeid93**
**What this PR does / why we need it**:
As @RafalSkolasinski truly commented on my last commit there was still problems in the previous merged notebook. I made some changes to the foldering and can confirm this version works fine on my local. Please review.

**Which issue(s) this PR fixes**:

Fixes #4080

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by seldondev** [comment]
Hi @saeid93. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) or [todo](https://github.com/orgs/todo/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/todo/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: examples/triton_gpt2/README.ipynb [hunk 1]
```diff
@@ -264,59 +272,45 @@
...
-      "\r\n",
-      "< x-envoy-upstream-service-time: 3\r\n",
-      "\r\n",
-      "< server: istio-envoy\r\n",
-      "\r\n",
-      "< \r\n",
-      "\r\n",
-      "* Connection #0 to host localhost left intact\r\n",
-      "{\"name\":\"gpt2\",\"versions\":[\"1\"],\"platform\":\"onnxruntime_onnx\",\"inputs\":[{\"name\":\"input_ids\",\"datatype\":\"INT32\",\"shape\":[-1,-1]},{\"name\":\"attention_mask\",\"datatype\":\"INT32\",\"shape\":[-1,-1]}],\"outputs\":[{\"name\":\"past_key_values\",\"datatype\":\"FP32\",\"shape\":[12,2,-1,12,-1,64]},{\"name\":\"logits\",\"datatype\":\"FP32\",\"shape\":[-1,-1,50257]}]}"
+      "*   Trying ::1:32000...\n",
```
[the remark this answers is not in the document]
Comment (axsaucedo) [reply examples/triton_gpt2/README.ipynb]: Perfect - nice oen

**Comment by axsaucedo** [comment]
@saeid93 it seems there are some lint issues, we use formatters for the notebooks as well so running those with the same versions should fix it

**Comment by saeid93** [comment]
@axsaucedo I couldn't find the lint issue, could you please have a look and let me know where should I fix?

**Comment by saeid93** [comment]
thanks for the explanation, hopefully, it'll work this time.

**Comment by axsaucedo** [comment]
NIce one @saeid93 - merging
